src/components/VmConsoles/LoadingMock.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
This would require changes to our test environment since file like `tests/tests/LoadingMock.test.js` would be required.
To fix, the component is moved to `VmConsoles.fixture.js`.